cmd/backup-agent/main.go

```diff
@@ -33,7 +33,7 @@ func main() {
 		fmt.Println(version.Version)
 		os.Exit(0)
 	}
-	ctrl.SetLogger(zap.New())
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
This makes the logs human readable, but we should probably add some sort of switch so that users can use full structured logs in production.